database/migrations/2021_02_25_175000_create_booking_categories_table.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateBookingCategoriesTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('booking_categories', function (Blueprint $table) {
+            $table->id();
+            $table->string('label');
```

Comment:
I prefer to use `name` instead of `label` here.

Reply:
@wolfrednicolas can you rename all labels to names?

The reason I'm using label because name and label is usually different thing. $user->name is `John` and user->label might be `John Thomas - Director of Marketing` (Labels are used for text representation of resource).

Reply:
OK, I tend to prefer the terminology of `title` and `name` and making a distinction between those.

With users, I separate out `first_name` and `last_name` although the existing app uses `name` for `first_name` and `name_last` for `last_name`. We'll have to make those updates throughout the codebase in all the packages during this refactoring.